x/oracle/keeper/reward.go

```diff
@@ -57,6 +57,7 @@ func (k Keeper) RewardBallotWinners(
 
 	// distribute rewards
 	var distributedReward sdk.Coins
+
 	for _, winner := range ballotWinners {
```

Comment:
here we had a bug, we need to iterate over slice, because in the loop we are updating distribution state.